packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
     sdk: flutter
   google_maps_flutter_android: ^2.1.10
   google_maps_flutter_ios: ^2.1.10
-  google_maps_flutter_platform_interface: ^2.2.1
+  google_maps_flutter_platform_interface: ^2.4.0
   google_maps_flutter_web: ^0.5.0
```

Comment:
Ideally, web should be released first, and then this dependency bumped to `^0.5.2`.

In practice, our tooling should be able to support this, but users may end up in an inconsistent state (that they can fix with a `flutter clean`, however).

WDYT @stuartmorgan, can this two packages be released at the same time? (Should they?)

Reply:
They should not be; the implementation should be split out as another PR so that we can do the version update. We always want to do that for new features so that we don't have the app-facing package updating saying "You can now do X!" and then when people update only the app-facing package (which is common) trying to do X doesn't actually work. (We forget sometimes, and we actually just had a bug where we forgot and a client got an unimplemented exception when trying to use a new feature.)

There are some non-feature cases where we can combine all the non-interface packages into one commit, but for features it's usually a chain of three, not two.